console/types/group/src/lib.rs

```diff
@@ -91,10 +91,6 @@ impl<E: Environment> Group<E> {
 
     /// Returns `self / COFACTOR`.
     pub fn div_by_cofactor(&self) -> Self {
-        // (For advanced users) The cofactor for this curve is `4`. Thus doubling is used to be performant.
-        // See unit tests below, which sanity check that this condition holds.
-        debug_assert!(E::Affine::cofactor().len() == 1 && E::Affine::cofactor()[0] == 4);
```

Comment:
This check was in both `mul_by_cofactor` and `div_by_cofactor`. However, only `mul_by_cofactor` is using this `double().double()` trick. This was a copypasta.